x-pack/plugin/otel-data/src/main/resources/component-templates/metrics-otel-fixed@mappings.yaml

```diff
@@ -0,0 +1,19 @@
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: |
+    Default mappings that can't be changed by users for
+    the OpenTelemetry metrics index template installed by x-pack
```

Comment:
It's the reverse. Users should be able to customize the dynamic mappings. That's why this component template needs to be imported last. This is because for dynamic templates, the first one that matches wins. Therefore, users need to be able to put a dynamic template before this in order to be able to customize how attributes are mapped dynamically. We have a test for this scenario:

https://github.com/elastic/elasticsearch/blob/3b9150dbb064a4afe3eb3602e9478d15eb72e409/x-pack/plugin/otel-data/src/yamlRestTest/resources/rest-api-spec/test/20_metrics_tests.yml#L112-L148

So I think all is correctly implemented, but we'll need  to change the description, and probably the name of the dynamic template. Suggestion: `ecs-tsdb@mappings` (since the ip path match is coming from ECS).